README.rst

```diff
@@ -36,17 +37,18 @@ This is easily achieved by downloading
 or individual libraries can be installed using
 `circup <https://github.com/adafruit/circup>`_.
 
-.. todo:: Describe the Adafruit product this library works with. For PCBs, you can also add the
-image from the assets folder in the PCB's GitHub repo.
+This supports the common 4" and 5.65" ACeP displays.
+
+* [Waveshare 4"](https://www.waveshare.com/4.01inch-e-paper-hat-f.htm)
+* [Waveshare 5.65"](https://www.waveshare.com/5.65inch-e-paper-module-f.htm)
+* [Pimoroni 5.7"](https://shop.pimoroni.com/products/inky-impression-5-7) (Likely the same as 5.65".)
+* [Pimoroni 4"](https://shop.pimoroni.com/products/inky-impression-4)
 
-`Purchase one from the Adafruit shop <http://www.adafruit.com/products/>`_
 Installing from PyPI
 =====================
 .. note:: This library is not available on PyPI yet. Install documentation is included
    as a standard element. Stay tuned for PyPI availability!
```

Comment:
I thought we are doing everything on PyPi now, and that the cookie-cutter choices do that.

Reply:
This is still here in the cookiecutter, likely under the assumption that maybe community libraries won't publish (or at least that was my thought).  It may be worth removing this from the template, or at least doing so if it's an Adafruit library.

Reply:
I didn't put it on pypi because blinka displayio hasn't been updated to work with acep displays. I imagine it will be at some point though. That's why I left it here.